apps/trivium/src/trivium/test.rs

```diff
@@ -0,0 +1,411 @@
+use tfhe::prelude::*;
```

Comment:
can you add a Makefile target to run the tests of trivium please ? :)

Reply:
I'm adding one for Trivium and one for Kreyvium